mlperf_logging/package_checker/package_checker.py

```diff
@@ -265,6 +265,13 @@ def get_parser():
         action='store_true',
         help='Bypass failed RCP checks so that submission uploads'
     )
+    parser.add_argument(
+        '--bert_train_samples',
```

Comment:
might be better to use a more specific name that mentions "rcp" here, e.g. rcp_bert_use_train_samples, since the package checker is not just for rcp and it might become confusing to future user.